table-store/src/main/java/com/scalar/dl/tablestore/client/partiql/statement/GetHistoryStatement.java

```diff
@@ -0,0 +1,72 @@
+package com.scalar.dl.tablestore.client.partiql.statement;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.base.MoreObjects;
+import com.scalar.dl.genericcontracts.table.v1_0_0.Constants;
+import com.scalar.dl.tablestore.client.util.JacksonUtils;
+import java.util.List;
+import java.util.Objects;
+
+public class GetHistoryStatement extends AbstractJacksonBasedContractStatement {
+
+  private final String contractId;
+  private final JsonNode arguments;
+
+  private GetHistoryStatement(JsonNode arguments) {
+    this.contractId = Constants.CONTRACT_GET_HISTORY;
+    this.arguments = Objects.requireNonNull(arguments);
+  }
+
+  @Override
+  public String getContractId() {
```

Comment:
It's not directly related to this PR, but these statement classes look a bit redundant.
contracId and arguments, and their corresponding methods including toString and hashcode, can be defined in the superclass?